### ISSUE TITLE
Training loss just keep increasing

### ISSUE THREAD

**Issue by agilotra** [PatrickLib/captcha_recognition#1]
Model doesnot work. Training loss never stablizes. Here is the proof.

>> Step 440 run_train: loss = 15.94 (8.119 sec)
>> Step 450 run_train: loss = 36.65 (8.486 sec)
>> Step 460 run_train: loss = 102.19 (8.480 sec)
>> Step 470 run_train: loss = 317.27 (9.262 sec)
>> Step 480 run_train: loss = 801.44 (8.208 sec)
>> Step 490 run_train: loss = 2089.72 (7.839 sec)
>> Step 500 run_train: loss = 5845.62 (7.887 sec)
>> 2017-08-26 13:12:49.057376 Saving in ./captcha_train/captcha
>> Step 510 run_train: loss = 13428.68 (7.818 sec)
>> Step 520 run_train: loss = 25587.19 (9.801 sec)
>> Step 530 run_train: loss = 59853.98 (7.978 sec)
>> Step 540 run_train: loss = 108795.05 (8.611 sec)
>> Step 550 run_train: loss = 190373.88 (8.281 sec)
>> Step 560 run_train: loss = 308137.38 (7.726 sec)

Configuration:
IMAGE_HEIGHT = 65
IMAGE_WIDTH = 190
CHAR_SETS = 'abcdefghijklmnopqrstuvwxyz'
CLASSES_NUM = len(CHAR_SETS)
CHARS_NUM = 8

**Comment by PatrickLib** [comment]
IMAGE_HEIGHT = 65, IMAGE_WIDTH = 190, CHARS_NUM =8 also can work fine. do you change loss function in captcha_models.py or image decode in captcha_input.py? what's kind of your captcha generate ? you can redownload the master code, using generate_default and try again. have a good luck!

**Comment by agilotra** [comment]
Attaching few of captchas i was trying to decode
![ablafe_1437502_1501587421](https://user-images.githubusercontent.com/10847377/31607708-09d38940-b28b-11e7-853b-369caa81eab6.jpeg)
![ablded_1473985_1501586593](https://user-images.githubusercontent.com/10847377/31607710-0c891eb6-b28b-11e7-9ef5-d060edfe7403.jpeg)
![acisses_1468563_1501586650](https://user-images.githubusercontent.com/10847377/31607721-173714e4-b28b-11e7-8bac-3004572c781e.jpeg)
![acises_1439585_1501587037](https://user-images.githubusercontent.com/10847377/31607726-18448e8e-b28b-11e7-8cd3-106c19d8a6f4.jpeg)
![aciming_1473180_1501586600](https://user-images.githubusercontent.com/10847377/31607725-18064336-b28b-11e7-8411-97dad78d3e37.jpeg)
![acirter_1449470_1501586944](https://user-images.githubusercontent.com/10847377/31607729-193918d2-b28b-11e7-8421-6afc2174d2da.jpeg)

**Comment by skanger** [comment]
so do I, always loss , I can't train my data
